headers_test.go

```diff
@@ -596,6 +603,13 @@ func TestUnprotectedHeader_UnmarshalCBOR(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "int map key too large",
+			data: []byte{
+				0xa1, 0x3b, 0x83, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30,
```

Comment:
Same here. We want a complete _label -> value_ pair in the map.
```suggestion
				0xa1, 0x3b, 0x83, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x00,
```

Reply:
Hmm, when putting the original sequence in [cbor.me](https://cbor.me/?bytes=21((13.68.16.48.48.48.16.48.88.30.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.84.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.48.65.48) it tells me that the byte chunk represents a complete map, and that the tailing `0x00` is unused.

Reply:
Ah. My mistake. I counted it wrongly. Could you also revert the change at line 257?